scipy/signal/_signaltools.py

```diff
@@ -3109,7 +3109,10 @@ def resample(x, num, t=None, axis=0, window=None, domain='time'):
 
     As noted, `resample` uses FFT transformations, which can be very
     slow if the number of input or output samples is large and prime;
-    see `scipy.fft.fft`.
+    see `scipy.fft.fft`. In such cases, it can be faster to use the polyphase
+    method in `scipy.signal.resample_poly`. If you need to set the number
+    of samples in the resampled signal to a specific value, you can use
+    `scipy.signal.resample_poly` before using `resample`.
```

Comment:
Perhaps a formulation something like the following  would be little bit clearer. Also noting that the antialiasiing filters are different might be helpful. E.g.:

```rst
In such cases, it can be faster to first downsample a signal of length `n` with 
:func:`~scipy.signal.resample_poly` by a  factor of ``n//num`` before using `resample`.  
Note that this approach changes the characteristic of the antialiasing filter.
```